README.md

```diff
@@ -442,7 +443,7 @@ __Tip 2:__ If you use one of unsupported CI providers ([here is list of supporte
 
 ### Repository adapter (How to set up 3 of 3)
 
-#### When you NOT set global variable `KNAPSACK_PRO_REPOSITORY_ADAPTER` (default)
+#### When you do NOT set global variable `KNAPSACK_PRO_REPOSITORY_ADAPTER` (default)
```

Comment:
Do we mean here "When you should NOT" set this? If so, then I would say "should NOT" instead of "do NOT".

Reply:
No.

This is default, when you do not set the global variable.

Reply:
I'm not sure if I understand.

What I'm struggling with is, as a reader, I don't know what exactly you mean by saying "when you do NOT set..."

Reply:
Maybe the header could be rephrased to:

By default `KNAPSACK_PRO_REPOSITORY_ADAPTER` environment variable is undefined

Reply:
Okay, sounds good

Reply:
Fixed in https://github.com/KnapsackPro/knapsack_pro-ruby/commit/1ba4f4bf6aa7836391d477032ed256edc540a034